source/Character.hx

```diff
@@ -233,22 +233,10 @@ class Character extends FlxSprite
 				animation.addByPrefix('idle', "Pico Idle Dance", 24);
 				animation.addByPrefix('singUP', 'pico Up note0', 24, false);
 				animation.addByPrefix('singDOWN', 'Pico Down Note0', 24, false);
-				if (isPlayer)
```

Comment:
The isPlayer code is unnecessary because there's already code that flips the singLEFT/RIGHT animations